utilities_common/helper.py

```diff
@@ -35,6 +37,13 @@ def get_port_acl_binding(db_wrap, port, ns):
     return acl_tables
 
 
+def validate_interface_name_length(iface_name):
+    """
+    Verify that interface name length does not exceed IFNAMSIZ
+    """ 
+    return True if len(iface_name) >= IFNAMSIZ else False
```

Comment:
@arfeigin please have:
```python
if len(iface_name) == 0:
    return False

return True if len(iface_name) < IFNAMSIZ else False
```
**return value:**
True - name is valid
False - name is invalid

**details:**
https://www.gnu.org/software/libc/manual/html_node/Interface-Naming.html

`Constant: size_t IFNAMSIZ`
This constant defines the maximum buffer size needed to hold an interface name, including its terminating zero byte.

Reply:
good catch! fixed